.github/workflows/deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy pages
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2.1.5
+        with:
+          node-version: '14'
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: |
+          cd ./showcases/2021-06/
+          yarn install
+      - name: Build application
+        run: |
+          cd ./showcases/2021-06/
+          yarn build
+          cp dist/index.html dist/404.html
+      - name: Deploy pages
+        uses: crazy-max/ghaction-github-pages@v2.3.0
+        with:
+          build_dir: ./showcases/2021-06/dist
```

Comment:
Will we be building all the presentations or only the most recent one?

Reply:
I need to think about how to get many pages to build at once

Reply:
I've finally got it to build once (apologies if you got an email for every push)

https://quantumblacklabs.github.io/kedro-community/

Do you have any idea how I can host many? At the moment I think this will only support one at a time